docs/guides/file-system-based-automated-bundle-generation.md

```diff
@@ -185,6 +185,14 @@ Once generated, all server entrypoints will be imported into a file named `[Reac
 
 *Note: If specifying separate definitions for client and server rendering, please make sure to delete the generalized `ComponentName.jsx` file.*
 
+### Integrating auto-bundling into CI workflows
+
+Currently, ReactOnRails contains conditional logic that checks for the existence of generated entrypoint files whenever `react_component` or one of its derivative methods are called. This conditional logic is used to determine the relative file path to provide `append_javascript_pack` as an argument.
+
+This means that `rake react_on_rails:generate_packs` or its programmatic equivalent must be run as a prerequisite to any sort of test or spec that would result in `react_component` or one of its derivative methods being called, even if the generated entrypoint files have already been bundled in a previous workflow/job.
+
+Caching of the generated entrypoints would likely also resolve this issue.
```

Comment:
we need to mention the react on rails spec helper will do this automatically

thus, it should be used and not `compile: true` in the shakapacker setup.

Reply:
While the `TestHelper` does generate packs if they are stale, it only uses mtime to check & doesn't have a digest strategy available, like Shakapacker does.

Before we push users to switch, would you like me to investigate having `TestHelper` use Shakapacker's stake-checking strategies so that we can reuse code & specs?